cmd/gvm/gvm.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/andrewkroh/gvm"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+const usage = `gvm is a Go version manager. gvm installs a Go version and prints
+the commands to configure your environment to use it. gvm can only install
+binary versions of Go from https://golang.org/dl/. Below are examples for
```

Comment:
I guess it's not limited to "binary versions" now.

Reply:
It's still limited to binaries for now. Due to the many refactorings I'd rather want to add compile support in another PR.